components/telemetry-operator/README.md

```diff
@@ -0,0 +1,69 @@
+# Telemetry Operator
+
+## Overview
+
+The telemetry operator contains a logging controller that generates a Fluent Bit configuration from one or more `LoggingConfiguration` custom resources. The controller ensures that all Fluent Bit pods run the current configuration by deleting pods after the configuration has changed. Find all CRD attributes [here](api/v1alpha1/loggingconfiguration_types.go) and an example [here](config/samples/telemetry_v1alpha1_loggingconfiguration.yaml).
+
+Creating Fluent Bit pods is for now out of scope of the operator and an existing Fluent Bit DaemonSet is expected.
+
+The generated ConfigMap (`logging-fluent-bit-sections` in the `kyma-system` namespace by default) has to be mounted to the Fluent Bit pods and consumed by an `@INCLUDE` statement in an existing [configuration file](https://docs.fluentbit.io/manual/administration/configuring-fluent-bit/configuration-file). File references and environment variables are available in an additional ConfigMap or Secret.
+
+See the flags the configure all ConfigMaps, Secret and DaemonSet names in [main.go](main.go).
+
+The operator has been bootstrapped with [Kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) 3.1.0. Additional APIs can also be [added by Kubebuilder](https://book.kubebuilder.io/cronjob-tutorial/new-api.html).
+
+## Development
+
+### Prerequisites
+- Install [kubebuilder 3.1.0](https://github.com/kubernetes-sigs/kubebuilder) which is the base framework for this controller
+- Install [kustomize](https://github.com/kubernetes-sigs/kustomize) which lets you customize raw, template-free `yaml` files during local development
+- Install [Golang 1.16](https://golang.org/dl/) or newer (for local execution)
+- Install [Docker](https://www.docker.com/get-started)
+
+### Usage
+
+You can do the following:
```

Comment:
```suggestion
```suggestion

### Available Commands

For development, you can use the following commands:

``````